src/views/ProtectionHome.vue

```diff
@@ -129,8 +131,27 @@ export default class ProtectionHome extends BaseComponent {
     }
   ];
 
-  positionFilterFunction(row: ViewProtectedLiquidity) {
-    return true;
+  positionFilterFunction(row: ViewGroupedPositions) {
+    const now: number = Date.now() / 1000;
+    const isFullyProtected: boolean = row.fullCoverage - now < 0;
+
+    const selectedIndex = this.dropDownFilters[0].selectedIndex;
+    if (selectedIndex == 1) {
+      row.collapsedData = row.collapsedData.filter(
+        x => x.fullCoverage - now < 0
```

Comment:
Let's adopt a more consistent approach by using some of the DayJS methods we've been using elsewhere. 

An example
```
  const now = dayjs();
  const availableClaims = this.lockedBalancesArr
      .filter(balance => dayjs.unix(balance.expirationTime).isBefore(now))
```